devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy

```diff
@@ -67,6 +67,9 @@ metadata {
 		
 		//XLSmart
 		fingerprint profileId: "C05E", manufacturer: "GLEDOPTO", model: "GL-B-001Z", deviceJoinName: "XLSmart E14 RGBW Light Bulb"
+
+		// GLEDOPTO
```

Comment:
Put this in alphabetical order by manufacturer. It's less likely to get conflicts between multiple PRs than when new ones are added to the end.